src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Text_Format.txt

```diff
@@ -43,12 +43,6 @@
 >> Text(0, "short day") 
 "00ort 30a99"
 
->> Text(DateTime(1983, 6, 3, 2, 53, 9, 0), "dddd, mmm "" dd, yyy") 
-"Friday, Jun "" 03, 1983"
-
->> Text(30470, "DDDD, MMM "" DD, YYY") 
-"Friday, Jun "" 03, 1983"
-
 >> Text(DateTime(1983, 6, 3, 2, 53, 9, 0), "dddd, mmm """" dd, yyy") 
```

Comment:
I see that those tests have been moved to 'Text_Format_PowerFxV1Compat.txt' - it would be good to have them in 'Text_Format_PowerFxV1CompatDisabled.txt' with the existing result.

Reply:
Add more test to disabled file